gulp/util/create-config.js

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2015 Palantir Technologies, Inc. All rights reserved.
+ */
+"use strict";
+
+module.exports = function createConfig (project) {
```

Comment:
I am a micro-commit maniac, so you check [db59e23](https://github.com/palantir/blueprint/pull/381/commits/db59e23a0fcec2956310273dbc8239895ea83387). It strictly refactors the `createConfig` function out of that file. 

I thought that code would need to be shared but in the end it wasn't. This change is not needed for this PR, I almost took it up but kept it in as a last-minute decision because it cleaned up `karma.js` significantly. It should definitely be named `karma-config.js`. 

Can add a revert commit to the PR if you'd like to keep the code within `karma.js`. Just give me your final call on revert vs rename.